core/src/main/java/com/graphhopper/storage/GraphEdgeIdFinder.java

```diff
@@ -19,16 +19,13 @@
 
 import com.graphhopper.coll.GHIntHashSet;
 import com.graphhopper.routing.util.EdgeFilter;
-import com.graphhopper.util.shapes.Polygon;
 import com.graphhopper.storage.index.LocationIndex;
 import com.graphhopper.storage.index.QueryResult;
 import com.graphhopper.util.BreadthFirstSearch;
 import com.graphhopper.util.EdgeIteratorState;
 import com.graphhopper.util.PointList;
-import com.graphhopper.util.shapes.BBox;
-import com.graphhopper.util.shapes.Circle;
-import com.graphhopper.util.shapes.GHPoint;
-import com.graphhopper.util.shapes.Shape;
+import com.graphhopper.util.shapes.Polygon;
```

Comment:
Can be removed because the next line should cover it as well. But I would try to avoid `*` and add separate import statements to use the same style in the project.

Reply:
intellij defaults to use star imports as soon as there are 5 of the same package. this means sometimes its going back and forth between listing all imports and doing star import. to me this threshold seems a bit low and we could set it to 10 (or infinity even). but yeah not a big issue really...